src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/NotificationReferenceBuilder.java

```diff
@@ -165,4 +165,17 @@ public static String referenceForDefendant(String claimReferenceNumber, String o
             return reference(TEMPLATE, DEFENDANT, otherParty.toLowerCase(), claimReferenceNumber);
         }
     }
+
+    public static class ClaimantResponse {
```

Comment:
We don't need this new class as we already have similar static inner class in same file called `ClaimantResponseSubmitted`.

Reply:
its gone now